src/Microsoft.AspNet.Identity.AspNetCoreCompat/Microsoft.AspNet.Identity.AspNetCoreCompat.csproj

```diff
@@ -4,7 +4,7 @@
 
   <PropertyGroup>
     <Description>A compatibility layer for sharing identity databases between Microsoft.AspNet.Identity.EntityFramework and Microsoft.AspNetCore.Identity.EntityFrameworkCore.</Description>
-    <VersionPrefix>0.3.0</VersionPrefix>
+    <VersionPrefix Condition="'$(ExperimentalProjectVersionPrefix)' != ''">$(ExperimentalProjectVersionPrefix)</VersionPrefix>
```

Comment:
Why the experimental project version prefix? I'd assume that we would never decide during a patch update to designate a pre-release package as rtm. In fact, in the past, we do not ship patch updates for pre-release packages at all.

Reply:
Yeah, idk. @Tratcher ? We shipped 0.3.0, to nuget.org so I assume we should also ship 0.3.1.

Reply:
cc @Eilon would we want to ship updates for pre-release packages in 2.0.x patches? We decided not to do this in 1.x patches.

Reply:
@haok

Reply:
I don't think we need to update this package, it never was officially released and is more of a prototype/sample anyways.

Reply:
Yeah in the past we didn't ship patches of these, so no point starting now. Less stuff to deal with.

Reply:
Ok. I'll mark this package as "noship" in the packages.csv list. That is what ultimately determines what will ship.